README.md

```diff
@@ -333,48 +339,54 @@ public interface ExampleHandler {
   // ...
 }
 ```
-You can declare your own methods that will be called in `onInvoke()` method of basic ManualHandler.
+You can declare your own methods that will be called in `onInvoke()` method of
```

Comment:
I think that we should avoid this statement as it reveals unneeded implementation details. And instead write the usages of JSTPHandler at the beginning of this section.